smart-contracts/hardhat.config.js

```diff
@@ -30,7 +30,7 @@ const { getHardhatNetwork } = require('./helpers/network')
 const settings = {
   optimizer: {
     enabled: true,
-    runs: 200,
+    runs: 20,
```

Comment:
This does not leave us a lot of margin :/ wdyt?

Reply:
so from trial and error, we can get as low as 140 but ofc thats not a viable solution. Filed an issue for now